### ISSUE TITLE
Support translation to multiple languages in parallel

### ISSUE THREAD

**Issue by wichert** [WeblateOrg/weblate#197]
In some situations it would be very useful to be able to see and modify translations for multiple languages at the same time. For example when working on dialects (nl_NL and nl_BE, or zh_CN and zh_TW), or when working on similar languages (no and nn) this would be a huge timesaver.
##

**Comment by BhaaLseN** [comment]
Was about to create a new issue for this (if I didn't find one), so I ended up here.

I see two scenarios that we have, and secondary languages can actually help with one of those.

The first one would be to compare multiple languages in case a check triggers (such as "trailing period" or whatever). Sometimes we want the period there, but sometimes it is a mistake and we don't; where it makes sense to see all translations for that given key to compare (like, do all but one have the period, or is it a single stray?)

The second would be merely a way of work optimization for our translators. Some work on two or even more languages in total, and they asked if they could edit them together.

I can solve the first one using secondary languages since we don't have that many languages for most of the projects; but the second one is still in need of a better solution.

**Comment by ws-hex** [comment]
The ability to edit translations within the "Other languages" tab would be a great addition.

We imported many projects with existing translations. So many failed checks for trailing white space, full stops, etc are taking orders of magnitude longer to fix than it needs to.

**Comment by swurzinger** [comment]
Zen mode is great to speed up translation, but allowing to add other languages in that view (currently only origin language and translated language) is something we are missing. For example czech and slovak languages seem to be very similar and there's just a single person maintaining both translations at the same time. Same applies to language variants, for example de_AT, de_CH, de_DE (austrian/swiss/germany - german).

**Comment by nijel** [comment]
You can already display other languages, see https://docs.weblate.org/en/latest/user/profile.html?highlight=secondary%20languages#secondary-languages

**Comment by swurzinger** [comment]
Thanks, this is indeed a nice feature - especially in Zen-Editor mode - in full-editor mode the "other languages" list below provides similar functionality. Unfortunately the secondary language fields are read-only and cannot be edited. In comparison to the "other languages" list there's no no edit button either to jump to a page that allows editing.

WebLate' behavior of always jumping to the next translation after clicking the "Save" button makes multi-language translations even more complicated - currently this needs the following steps in full-editor mode:

1. edit translation in first language (e.g. cs_CZ) and click "Save"
2. click back navigation button to view the just translated string again
3. identify the second language (e.g. sk_SK) in the "other languages" list and click "Edit"
4. edit translation in second language and click "Save"
5. perform steps 2 (optional) and 3 again to continue with the first language

**Comment by Belorus** [comment]
This would be a great feature. We as developers add translations key by key and have all languages at edit-time. 

Please add this

**Comment by marknitek** [comment]
This would be a great addition. For supporting language variants like de-CH and de-DE / de-AT i would like to keep the translation the same (100%) is that possible out of the box? For example can we use automatic translation from another language?

**Comment by Geeyun-JY3** [comment]
FYI, Crowdin has released a Multilingual View several months ago: https://blog.crowdin.com/2022/01/11/what-is-new-at-crowdin-december/#multilingual-editor-view-manage-large-projects-easier

**Comment by Geeyun-JY3** [comment]
Crowdin’s Multilingual View comes with a grid view now: https://blog.crowdin.com/2022/08/03/what-is-new-at-crowdin-july-2022/#display-more-information-using-editor-grid-view

**Comment by Roelsoft** [comment]
All of us speak both English and Dutch and if we change the English translation we can then also change the Dutch translation to match. Or vice versa. Now you have to first click on English and then click to the Dutch language. Even if both input boxes are right there but one is read-only. Seems really silly to me considering the only change is to not make it read-only to make this available. And if somebody is authorized to translate multiple languages I don't see any reason why they shouldn't be able to do that in one window. Which is the best UX option.
